### PR TITLE
Configs for Multus CNI, SR-IOV device plugins and ancillary components

### PR DESCRIPTION
**Pull request by dougbtv**
A number of components to have images built upstream especially related to the support of functionality which allows the attachment of multiple network interfaces to pods.

### CODE REVIEW

**Comment by smarterclayton** [comment]
@petr-muller if you can review

**Comment by petr-muller** [comment]
> @petr-muller if you can review

sorry I was busy yesterday prepping for the demo, I'll have a look today

File: ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+canonical_go_repository: github.com/openshift/containernetworking-plugins
+images:
+- dockerfile_path: openshift/containernetworking-plugins/images/supported/Dockerfile.openshift
+  from: base
+  to: container-networking-plugins-supported
+- dockerfile_path: openshift/ose-containernetworking-plugins/images/unsupported/Dockerfile.openshift
```
[the remark this answers is not in the document]
Comment (fepan) [reply ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml]: Ah yes. Fixed!

[the remark this answers is not in the document]
Comment (fepan) [reply ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml]: Oh right. Those should just be Dockerfile and not Dockerfile.openshift, we do not have upstream builds of this repo. Updated the file path.

**Pull request merged**